python/rola/utils/ret.py

```diff
@@ -0,0 +1,33 @@
+from radix_engine_toolkit import (
+    derive_virtual_identity_address_from_public_key,
+    PublicKey,
+    Address,
+    Curve,
+)
+
+from rola.models.challenge import ChallengeType
+from rola.models.signed_challenge import SignedChallenge
+
+
+def derive_address(network_id: int, signed_challenge: SignedChallenge) -> Address:
+    public_key_int_list = [byte for byte in signed_challenge.proof.public_key]
+    if signed_challenge.challenge_type == ChallengeType.PERSONA:
+        return derive_virtual_identity_address_from_public_key(
+            network_id=network_id, public_key=PublicKey.ED25519(public_key_int_list)
+        )
+    elif signed_challenge.challenge_type == ChallengeType.ACCOUNT:
+        if signed_challenge.proof.curve == Curve.ED25519:
+            return derive_virtual_identity_address_from_public_key(
```

Comment:
Should we use `derive_virtual_identity_address_from_public_key` or `derive_virtual_account_address_from_public_key`?

Reply:
good point. it should be derive_virtual_account_address_from_public_key